polars/polars-lazy/polars-plan/src/dsl/mod.rs

```diff
@@ -1493,6 +1493,25 @@ impl Expr {
         })
     }
 
+    #[cfg(feature = "cutqcut")]
+    pub fn nqcut(
+        self,
+        nbins: usize,
```

Comment:
Can you underscore `n_bins`? :)

No worry, I don't want it longer than that.

Reply:
I think I can live with that ;-)